trustyai-service-operator/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/trustyai/trustyai-service-operator
+  newTag: latest
```

Comment:
We should probably change this tag to a fixed one.

Reply:
done